sunpy/net/vso/__init__.py

```diff
@@ -6,4 +6,4 @@
 # Delete this in 0.9
 from sunpy.net.vso.vso import search, get
 
-__all__ = ['VSOClient', 'InteractiveVSOClient']
+__all__ = ['VSOClient', 'InteractiveVSOClient', 'QueryResponse', 'search', 'get']
```

Comment:
I don't think `search` and `get` should be added here, they are deprecated anyway.